### PR TITLE
Search filter

### PR DESCRIPTION
**Pull request by Lydias303**
Styled the search filter. It filters by all of the trail keys.

### CODE REVIEW

File: app/components/trail-map.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+
+  map: null,
+  classNames: ['trail-map'],
+
+  didInsertElement: function () {
+
+    L.mapbox.accessToken = 'pk.eyJ1IjoibHlkaWFzMzAzIiwiYSI6ImM0WG9rY28ifQ.bM1Nx1fsDmbAFLVP1f9Img';
```
Comment (EmilyMB) [inline app/components/trail-map.js:12]: Maybe hide this?

**Comment by EmilyMB** [comment]
:heart: the search functionality...as discussed would prefer a slightly different appearance.

File: config/environment.js [hunk 2]
```diff
@@ -20,12 +20,14 @@ module.exports = function(environment) {
   };
 
   if (environment === 'development') {
-    // ENV.APP.LOG_RESOLVER = true;
-    // ENV.APP.LOG_ACTIVE_GENERATION = true;
-    // ENV.APP.LOG_TRANSITIONS = true;
-    // ENV.APP.LOG_TRANSITIONS_INTERNAL = true;
-    // ENV.APP.LOG_VIEW_LOOKUPS = true;
-  }
+     ENV.contentSecurityPolicy = {
```
Comment (cluhring) [inline config/environment.js:23]: I particularly like this bit here - less errors = :) Chris

**Comment by cluhring** [comment]
I didn't see a lot of tests for this, but your code is quite clever.

**Comment by cluhring** [comment]
What does the hound think of this?

**Pull request merged**